src/redis_request.cc

```diff
@@ -28,6 +28,7 @@ Status Request::Tokenize(evbuffer *input) {
           if (pipeline_size > 128) {
             LOG(INFO) << "Large pipeline detected: " << pipeline_size;
           }
+          if (line) continue;
```

Comment:
```suggestion
          if (line) {
            free(line);
            continue;
          }
```

Reply:
yeah, we should free the line here if not null.